composer.json

```diff
@@ -40,7 +40,8 @@
         "burgov/key-value-form-bundle": "to edit assoc multivalue properties. require version 1.0.*"
     },
     "conflict": {
-        "phpcr/phpcr-shell": "<1.0.0-beta1"
+        "phpcr/phpcr-shell": "<1.0.0-beta1",
+        "symfony/framework-bundle": ">=2.6.0,<2.6.6"
```

Comment:
did 2.4 and 2.5 get the fix? I assume not?

in this case maybe just do the following in the require: ``~2.3.27,~2.6``